groclient/client.py

```diff
@@ -958,60 +965,76 @@ def get_df(self, show_revisions=False, show_available_date=False, index_by_serie
                If set, the dataframe will have additional columns with names of entities.
                Note that this will increase the size of the dataframe by about 5x.
             compress_format: boolean, optional
-               If set, each series will be compressed to a single column in the dataframe, with the end_date column 
-               set as the dataframe inde. All the entity names for each series will be 
+               If set, each series will be compressed to a single column in the dataframe, with the end_date column
+               set as the dataframe inde. All the entity names for each series will be
                placed in column headers.
-
+            async_mode: boolean, optional
+                If set, it will make :meth:`~get_data_points` requests asynchronously.
+                Note that if you're using Jupyter Notebook, you will need to find a Tornado workaround
```

Comment:
mention the actual  workaround... e.g. "Note that when running in a Jupyter Ipython notebook with async_mode, you will need to use `nest_asyncio` module "

Reply:
updated